docs/docfx.json

```diff
@@ -23,7 +23,10 @@
           "src": "../src"
         }
       ],
-      "dest": "api"
+      "dest": "api",
+	  "properties": {
+        "TargetFramework": "netstandard1.6"
```

Comment:
Unless I'm missing something, we don't have different public APIs between ```net45``` and ```netstandard```. So I chose the later.